src/opendr/perception/object_detection_2d/retinaface/Makefile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020-2021 OpenDR European Project
```

Comment:
I am not sure that we need a second license. We can skip license tests for this file. We can add a comment if we have modified this file.